### PR TITLE
Added tables for allergies

### PR DESCRIPTION
**Pull request by MaksimTsedov**

### CODE REVIEW

File: src/Schema/Schema.sqlproj [hunk 1]
```diff
@@ -69,6 +69,10 @@
   <ItemGroup>
     <Build Include="Tables\UsersLogin.sql" />
     <Build Include="Tables\Roles.sql" />
+    <Build Include="Tables\Allergies.sql" />
```
Comment (Anastasiia37) [inline src/Schema/Schema.sqlproj]: Please, do one more directory for your tables

File: src/Schema/Tables/Allergies.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[Allergies]
+(
+	[AllergyId] INT NOT NULL PRIMARY KEY IDENTITY, 
+    [Pathogen] NVARCHAR(200) NOT NULL, 
+    [IsDeleted] BIT NOT NULL
```
Comment (Anastasiia37) [inline src/Schema/Tables/Allergies.sql]: We decided not to do Soft Delete in our base, or we don`t?

File: src/Schema/Tables/Allergies.sql [hunk 3]
```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[Allergies]
+(
+	[AllergyId] INT NOT NULL PRIMARY KEY IDENTITY,
```
Comment (Anastasiia37) [inline src/Schema/Tables/Allergies.sql]: IDENTITY(1, 1)

Comment (Anastasiia37) [inline src/Schema/Tables/Allergies.sql]: And PRIMARY KEY we should write in CONSTRAINT as in Anton`s example

File: src/Schema/Tables/Allergies.sql [hunk 4]
```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[Allergies]
+(
+	[AllergyId] INT NOT NULL PRIMARY KEY IDENTITY, 
+    [Pathogen] NVARCHAR(200) NOT NULL, 
+    [IsDeleted] BIT NOT NULL
+	CONSTRAINT UC_Allergies UNIQUE (Pathogen)
```
Comment (Anastasiia37) [inline src/Schema/Tables/Allergies.sql]: Use [ColumnName] instead ColumnName

Comment (Anastasiia37) [inline src/Schema/Tables/Allergies.sql]: And, I think, like in Anton`s example we should add spaces:
( [Pathogen] )

Comment (Anastasiia37) [inline src/Schema/Tables/Allergies.sql]: And before CONSTRAINT - one empty line

File: src/Schema/Tables/AllergySymptom.sql [hunk 5]
```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[AllergySymptom]
+(
+	[AllergySymptomId] INT NOT NULL PRIMARY KEY IDENTITY,
```
Comment (Anastasiia37) [inline src/Schema/Tables/AllergySymptom.sql]: Primary key and Identity as above

File: src/Schema/Tables/AllergySymptom.sql [hunk 6]
```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[AllergySymptom]
+(
+	[AllergySymptomId] INT NOT NULL PRIMARY KEY IDENTITY, 
+    [PatientAllergyId] INT NOT NULL, 
+    [SymptomId] INT NOT NULL, 
+    [IsDeleted] BIT NOT NULL, 
+    CONSTRAINT [FK_AllergySymptom_To_PatientAllergy] FOREIGN KEY (PatientAllergyId) REFERENCES PatientAllergy(PatientAllergyId),
```
Comment (Anastasiia37) [inline src/Schema/Tables/AllergySymptom.sql]: One empty line before constraint

File: src/Schema/Tables/AllergySymptom.sql [hunk 7]
```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[AllergySymptom]
+(
+	[AllergySymptomId] INT NOT NULL PRIMARY KEY IDENTITY, 
+    [PatientAllergyId] INT NOT NULL, 
+    [SymptomId] INT NOT NULL, 
+    [IsDeleted] BIT NOT NULL,
```
Comment (Anastasiia37) [inline src/Schema/Tables/AllergySymptom.sql]: The same as above

File: src/Schema/Tables/AllergySymptom.sql (hunk 6 above)
Comment (Anastasiia37) [inline src/Schema/Tables/AllergySymptom.sql]: [FK_AllergySymptom_To_PatientAllergy] - without [] and naming as in Anton`s example:
FK_Table1_Table2

Comment (Anastasiia37) [inline src/Schema/Tables/AllergySymptom.sql]: Add [] for columns and spaces between ():
FOREIGN KEY ( [PatientAllergyId] ) REFERENCES PatientAllergy( [PatientAllergyId] )

File: src/Schema/Tables/AllergySymptom.sql [hunk 8]
```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[AllergySymptom]
+(
+	[AllergySymptomId] INT NOT NULL PRIMARY KEY IDENTITY, 
+    [PatientAllergyId] INT NOT NULL, 
+    [SymptomId] INT NOT NULL, 
+    [IsDeleted] BIT NOT NULL, 
+    CONSTRAINT [FK_AllergySymptom_To_PatientAllergy] FOREIGN KEY (PatientAllergyId) REFERENCES PatientAllergy(PatientAllergyId), 
+    CONSTRAINT [FK_AllergySymptom_To_Symptom] FOREIGN KEY (SymptomId) REFERENCES Symptoms(Id)
```
Comment (Anastasiia37) [inline src/Schema/Tables/AllergySymptom.sql]: The same

File: src/Schema/Tables/Symptoms.sql [hunk 9]
```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[Symptoms]
+(
+	[Id] INT NOT NULL PRIMARY KEY IDENTITY,
```
Comment (Anastasiia37) [inline src/Schema/Tables/Symptoms.sql]: In your previous table you use PatientAllergyId, AllergyId etc. And here you decide to use just Id. May be we will develop a common strategy in this question and will stick to it?

File: src/Schema/Tables/PatientAllergy.sql [hunk 10]
```diff
@@ -0,0 +1,11 @@
+﻿CREATE TABLE [dbo].[PatientAllergy]
+(
+	[PatientAllergyId] INT NOT NULL PRIMARY KEY IDENTITY, 
+    [PatientId] INT NOT NULL, 
+    [AllergyId] INT NOT NULL,
+	[Duration] NVARCHAR(25) NULL,
+    [Notes] TEXT NULL, 
+    [IsDeleted] BIT NOT NULL, 
+    CONSTRAINT [FK_PatientAllergy_To_Allergies] FOREIGN KEY (AllergyId) REFERENCES Allergies(AllergyId), 
+    CONSTRAINT [FK_PatientAllergy_To_Patient] FOREIGN KEY (PatientId) REFERENCES PatientInfo(PatientId)
```
Comment (Anastasiia37) [inline src/Schema/Tables/PatientAllergy.sql]: Why do you write PatientInfo(PatientId)? Do you know that Alexandr will use PatientId? Or may be he will just Id like in our schema?

File: src/Schema/Tables/PatientAllergy.sql [hunk 11]
```diff
@@ -0,0 +1,11 @@
+﻿CREATE TABLE [dbo].[PatientAllergy]
+(
+	[PatientAllergyId] INT NOT NULL PRIMARY KEY IDENTITY, 
+    [PatientId] INT NOT NULL, 
+    [AllergyId] INT NOT NULL,
+	[Duration] NVARCHAR(25) NULL,
```
Comment (Anastasiia37) [inline src/Schema/Tables/PatientAllergy.sql]: I don`t see duration in our schema

File: src/Schema/Tables/PatientAllergy.sql [hunk 12]
```diff
@@ -0,0 +1,11 @@
+﻿CREATE TABLE [dbo].[PatientAllergy]
+(
+	[PatientAllergyId] INT NOT NULL PRIMARY KEY IDENTITY, 
+    [PatientId] INT NOT NULL, 
+    [AllergyId] INT NOT NULL,
```
Comment (Anastasiia37) [inline src/Schema/Tables/PatientAllergy.sql]: May be you can use unique constraint for [PatientId] + [AllergyId] ?

File: src/Schema/Tables/PatientAllergy.sql [hunk 13]
```diff
@@ -0,0 +1,11 @@
+﻿CREATE TABLE [dbo].[PatientAllergy]
+(
```
Comment (Anastasiia37) [inline src/Schema/Tables/PatientAllergy.sql]: The same about formalization as in previous tables

File: src/Schema/Tables/Symptoms.sql [hunk 14]
```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[Symptoms]
+(
```
Comment (Anastasiia37) [inline src/Schema/Tables/Symptoms.sql]: The same about formalization as in previous tables

**Review by Anastasiia37 — COMMENTED** [review]
Also you don`t have scrypt with static data

File: src/Schema/Tables/Allergies/Allergies.sql [hunk 15]
```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[Allergies]
+(
+	[AllergyId] INT NOT NULL  IDENTITY, 
+    [Pathogen] NVARCHAR(200) NOT NULL,
```
Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Allergies/Allergies.sql]: Please format everything properly. You can use some online sql formatter sites or add vs extension if you will find some. And please turn on to see 'dots' without spaces.

File: src/Schema/Tables/Allergies/AllergySymptom.sql [hunk 16]
```diff
@@ -0,0 +1,12 @@
+﻿CREATE TABLE [dbo].[AllergySymptom]
+(
+	[AllergySymptomId] INT NOT NULL  IDENTITY, 
+    [PatientAllergyId] INT NOT NULL,
```
Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Allergies/AllergySymptom.sql]: and here

File: src/Schema/Tables/Allergies/PatientAllergy.sql [hunk 17]
```diff
@@ -0,0 +1,15 @@
...
+(
+	[PatientAllergyId] INT NOT NULL  IDENTITY, 
+    [PatientId] INT NOT NULL, 
+    [AllergyId] INT NOT NULL,
+	[Duration] NVARCHAR(25) NULL,
+    [Notes] TEXT NULL, 
+    [IsDeleted] BIT NOT NULL, 
+
+	CONSTRAINT PK_PatientAllergy PRIMARY KEY ([PatientAllergyId]), 
+    CONSTRAINT FK_PatientAllergy_To_Allergies FOREIGN KEY ([AllergyId]) REFERENCES Allergies([AllergyId]),
```
Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Allergies/PatientAllergy.sql]: and here

File: src/Schema/Tables/Allergies/Symptoms.sql [hunk 18]
```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[Symptoms]
+(
+	[Id] INT NOT NULL  IDENTITY, 
+    [Naming] NVARCHAR(200) NOT NULL,
```
Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Allergies/Symptoms.sql]: and here

File: src/Schema/Tables/Allergies/PatientAllergy.sql [hunk 19]
```diff
@@ -0,0 +1,15 @@
...
+    [AllergyId] INT NOT NULL,
+	[Duration] NVARCHAR(25) NULL,
+    [Notes] TEXT NULL, 
+    [IsDeleted] BIT NOT NULL, 
+
+	CONSTRAINT PK_PatientAllergy PRIMARY KEY ([PatientAllergyId]), 
+    CONSTRAINT FK_PatientAllergy_To_Allergies FOREIGN KEY ([AllergyId]) REFERENCES Allergies([AllergyId]), 
+    CONSTRAINT FK_PatientAllergy_To_Patient FOREIGN KEY ([PatientId]) REFERENCES PatientInfo([PatientId]), 
+    CONSTRAINT UK_PatientAllergy UNIQUE ([PatientId], [AllergyId])
+
```
Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Allergies/PatientAllergy.sql]: redundant line

File: src/Schema/Tables/Allergies/PatientAllergy.sql [hunk 20]
```diff
@@ -0,0 +1,15 @@
...
+	[PatientAllergyId] INT NOT NULL  IDENTITY, 
+    [PatientId] INT NOT NULL, 
+    [AllergyId] INT NOT NULL,
+	[Duration] NVARCHAR(25) NULL,
+    [Notes] TEXT NULL, 
+    [IsDeleted] BIT NOT NULL, 
+
+	CONSTRAINT PK_PatientAllergy PRIMARY KEY ([PatientAllergyId]), 
+    CONSTRAINT FK_PatientAllergy_To_Allergies FOREIGN KEY ([AllergyId]) REFERENCES Allergies([AllergyId]), 
+    CONSTRAINT FK_PatientAllergy_To_Patient FOREIGN KEY ([PatientId]) REFERENCES PatientInfo([PatientId]),
```
Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Allergies/PatientAllergy.sql]: it`s okay to omit 'to'.  - FK_PatientAllergy_Patient in all FK`s

File: src/Schema/Tables/Allergies/AllergySymptom.sql [hunk 21]
```diff
@@ -0,0 +1,12 @@
+﻿CREATE TABLE [dbo].[AllergySymptom]
+(
+	[AllergySymptomId] INT NOT NULL  IDENTITY,
```
Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Allergies/AllergySymptom.sql]: here two spaces before IDENTITY

File: src/Schema/Tables/Allergies/Allergies.sql [hunk 22]
```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[Allergies]
+(
+	[AllergyId] INT NOT NULL  IDENTITY,
```
Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Allergies/Allergies.sql]: here two spaces before IDENTITY

File: src/Schema/Tables/Allergies/PatientAllergy.sql [hunk 23]
```diff
@@ -0,0 +1,15 @@
+﻿CREATE TABLE [dbo].[PatientAllergy]
+(
+	[PatientAllergyId] INT NOT NULL  IDENTITY, 
+    [PatientId] INT NOT NULL, 
+    [AllergyId] INT NOT NULL,
+	[Duration] NVARCHAR(25) NULL,
+    [Notes] TEXT NULL,
```
Comment (anton-borodulin-softserve) [inline src/Schema/Tables/Allergies/PatientAllergy.sql]: @Anastasiia37 @Kropisha @smoukiDev @FSuhov @ArteIIo @DenyaAbramenkov @MaksimTsedov

Some
--Mountain
Is
---Placed 
here

Guys, please format SQL code as well as you are doing with c#. Auto formatters, visual dots in place of spaces for VS. You can use anything that will help you to achieve that.

**Review by anton-borodulin-softserve — COMMENTED** [review]
Please fix formatting.

**Pull request closed without merging**